Project/Source/Components/UI/ComponentImage.cpp

```diff
@@ -75,7 +76,7 @@ void ComponentImage::OnEditorUpdate() {
 
 void ComponentImage::Save(JsonValue jComponent) const {
 	jComponent[JSON_TAG_TEXTURE_TEXTUREID] = textureID;
-
+	jComponent[JSON_TAG_ENABLED] = IsActive();
```

Comment:
This should be saved at the base Component level.

Reply:
Fair enough